django_python3_ldap/utils.py

```diff
@@ -30,6 +30,11 @@ def clean_user_data(user_data):
     return user_data
 
 
+def sync_user_relations(user, ldap_data):
+    # do nothing by default
+    return ldap_data
```

Comment:
This should really just be ``pass``, since the return value is ignored by the calling function.